### PR TITLE
refactor(pywasi-rx): move `python-reactor-wasi` to main repo

### PR DESCRIPTION
**Pull request by Yohe-Am**
Moves the [python-wasi-reactor](https://github.com/metatypedev/python-wasi-reactor) repository in-tree.

#### Motivation and context

MET-404

#### Migration notes

__No user changes required.__

### Checklist

- [ ] The change come with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

### CODE REVIEW

File: ghjk.ts [hunk 1]
```diff
@@ -69,13 +76,147 @@ if (!Deno.env.has("NO_PYTHON")) {
...
+    let WASI_VFS_CLI_DL;
+    switch (Deno.build.os) {
+      case "linux":
+        WASI_SDK_DL =
+          `https://github.com/WebAssembly/wasi-sdk/releases/download/${WASI_SDK_VERSION}/${WASI_SDK_VERSION}.0-linux.tar.gz`;
+        WASI_VFS_CLI_DL =
+          `https://github.com/kateinoigakukun/wasi-vfs/releases/download/${WASI_VFS_VERSION}/wasi-vfs-cli-x86_64-unknown-linux-gnu.zip`;
+        break;
+      case "darwin":
+        WASI_SDK_DL =
```
Comment (zifeo) [inline ghjk.ts:117]: I wonder whether we cannot include a helper to pick the correct architecture within ghjk from the a list of artefact, wdyt?

File: libs/python-wasi-reactor/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# Python WASI reactor
+
+> Python WASI reactor is part of the
+> [Metatype ecosystem](https://github.com/metatypedev/metatype). Consider
+> checking out how this component integrates with the whole ecosystem and browse
+> the
+> [documentation](https://metatype.dev?utm_source=github&utm_medium=readme&utm_campaign=python-wasi-reactor)
```
Comment (zifeo) [inline libs/python-wasi-reactor/README.md:7]: can we move this to the doc (e.g. under the ecosystem) section?

**Review by zifeo — COMMENTED** [review]
Nice job, lovely to see ghjk enabling this!

**Comment by Yohe-Am** [comment]
Superseded by #687 .

File: ghjk.ts (hunk 1 above)
Comment (Yohe-Am) [reply ghjk.ts:117]: Yeah, definitely. A lot of the ports pull from github releases so I was thinking a general purpose github helper. Something along the lines of [eget](https://github.com/zyedidia/eget). It ought to easily allow one to specify adhoc ports inline in the ghjkfile. 

And a big part of it is making it easy to match over target triplets. Being tracked [here](https://github.com/metatypedev/ghjk/issues/7) and sort of [here](https://github.com/metatypedev/ghjk/issues/38).

**Pull request closed without merging**